.github/workflows/main.yml

```diff
@@ -1,6 +1,7 @@
 on:
   schedule:
    - cron: '0 2 * * *'
+  push:
 
 jobs:
   delete_old_archives:
```

Comment:
This is a copy/paste error from when I made the repo, might as well change it now...
```suggestion
  update_gitstats:
```

Reply:
Ah, I just changed the job name to `mathlib_stats` before noticing your suggestion.